rust/platform/triple_mappings.bzl

```diff
@@ -196,6 +196,36 @@ def abi_to_constraints(abi):
         # figure out how they're doing this
         return []
 
+def triple_to_struct(triple):
```

Comment:
This already exists in `@rules_rust//rust/platform/triple.bzl`. Perhaps that could be used instead?

Reply:
Done.